src/Destructurama.JsonNet.Tests/Approval/Destructurama.JsonNet.approved.txt

```diff
@@ -1,6 +1,6 @@
 namespace Destructurama
 {
-    public static class LoggerConfigurationAppSettingsExtensions
+    public static class LoggerConfigurationJsonNetExtensions
```

Comment:
@nblumhardt  I think this class was initially copied from other project. I rename it in light of major version bump. OK?